### ISSUE TITLE
Applicant Home Program List Font is Very Tiny on Mobile

### ISSUE THREAD

**Issue by gwendolyngoetz** [civiform/civiform#5689]
**Describe the bug**

On mobile, the font for the cards on the applicants programs and services page `/applicants/:applicant_id/programs` is comically tiny. I feel like it used to be larger, but can't remember for certain. You end up having to zoom in a lot to read the text, but then do a lot of horizontal scrolling to read everything.

**To Reproduce**

Go to the home page of the application. It'll take you to the list of phones

**Expected behavior**
Fonts should be larger on mobile sized devices and scale in a way that makes them both readable in size and without horizontal scrolling.

**Screenshots**

![Tiny Font](https://github.com/civiform/civiform/assets/195162/1264976c-337d-4bea-a9b3-132101e8891b)

**Smartphone (please complete the following information):**

- Device: LG V60 (The last of a truly great line, rip LG phones :ghost: )
- OS: Android Something
- Browser: Firefox mobile
- Version: 117+

**Comment by caitlinshk** [comment]
We may want to look into addressing this with the program images work in #5676 (thanks for calling out Gwen!)

**Comment by gwendolyngoetz** [comment]
Northstar redesign is in progress and already scales the font size.
